test/scripts/openshift-ci/setup-e2e-tests.sh

```
@@ -116,13 +115,13 @@ pushd $PROJECT_ROOT/python/kserve >/dev/null
   uv pip install timeout-sampler
 popd
 
-# Install autoscaler only if NOT using ODH operator (operator handles it)
-if [[ "$INSTALL_ODH_OPERATOR" == "false" ]]; then
+# Install autoscaler only if NOT using an operator (operator handles it)
+if [[ "$USE_OPERATOR" == "false" ]]; then
   $SCRIPT_DIR/deploy.cma.sh
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)